package/osx/make-package

```diff
@@ -169,6 +190,9 @@ if [ -n "${build_x86_64}" ]; then
    arch -x86_64 "${CMAKE}" --build . --target all -- ${MAKEFLAGS}
    info "Done!"
 
+   # restore JAVA_HOME
+   restore-java-home
```

Comment:
Should we use a `trap` or similar to ensure this cleanup gets called even if the script stops on error?